filament/include/filament/Options.h

```diff
@@ -293,6 +296,7 @@ struct DepthOfFieldOptions {
         MEDIAN
     };
     float cocScale = 1.0f;              //!< circle of confusion scale factor (amount of blur)
+    float cocAspectRatio = 1.0f;        //!< width/height aspect ratio of the circle of confusion (simulate anamorphic lenses)
```

Comment:
I guess this new value won't impact our current DoF settings. Do you think linking this attribute to the app could enhance our DoF visuals?